client/app/components/Input/InputCheckbox.jsx

```diff
@@ -60,7 +66,7 @@ export const InputCheckbox = (props: Props) => {
           }
           aria-label={label.replace(/<\/?[^>]+(>|$)/g, '')}
         />
-        <div className={css.checkboxLabel}>{renderHTML(label)}</div>
+        <div className={`${css.checkboxLabel} ${resourcesPageDisplay(resourcesPage)}`}>{renderHTML(label)}</div>
```

Comment:
It would be better to pass in the css as props instead of checking the via the window.body whether a class exists. You can also override CSS as well.